package.json

```diff
@@ -60,8 +62,8 @@
     "@rollup/plugin-commonjs": "11.1.0",
     "@rollup/plugin-node-resolve": "7.1.3",
     "@rollup/plugin-url": "5.0.0",
-    "@semantic-release/changelog": "5.0.1",
-    "@semantic-release/git": "9.0.0",
```

Comment:
something is wrong with your merges. Unless you want to downgrade this for some reason